### ISSUE TITLE
Alternatives to MathJax

### ISSUE THREAD

**Issue by thewindsofwinter** [thewindsofwinter/JHMC-scripts#26]
Find a better way to display MathJax. Reminder for me to work on this over the summer or next year.

**Comment by phultquist** [comment]
There's no reason for it to be client side rendered, and not an image. I was doing compusci, trying to display latex in markdown (which is how I write my compusci answers), and I found [this](https://stackoverflow.com/questions/11256433/how-to-show-math-equations-in-general-githubs-markdownnot-githubs-blog), which seems pretty good for what we're doing, although it adds another dependency on a server that we're counting to be up.

This would take away a huge headache... maybe you can host the tex 2 png software on our server that way you don't rely on another server being up.

**Comment by phultquist** [comment]
people keep sending me things/i keep finding things online but here's [Katex](https://katex.org), and honestly it looks really impressive

**Comment by phultquist** [comment]
Also i never showed you this email but fogel reccomended PreTeXt

**Comment by thewindsofwinter** [comment]
Yeah I think I received that email (forwarded?) a while back thanks for bringing it to my attention, I think revisions to JHMC scripts will probably happen most likely after Spring Break at the earliest thought.

**Comment by thewindsofwinter** [comment]
> people keep sending me things/i keep finding things online but here's [Katex](https://katex.org), and honestly it looks really impressive

Yeah that looks cool af
